R/ahr_blinded.R

```diff
@@ -111,11 +111,10 @@ ahr_blinded <- function(
   # Compute adjustment for information
   q_e <- ratio / (1 + ratio)
 
-  ans <- tibble(
+  tibble(
     event = sum(event),
     ahr = exp(-theta),
     theta = theta,
     info0 = sum(event) * (1 - q_e) * q_e
   )
-  return(ans)
```

Comment:
Personally I like the explicit return even though it is optional. And checking `git blame`, I'm pretty sure they existed before I started editing the code base. I'm happy to go along with the consensus though

Reply:
I don't have a strong opinion here and can definitely use an explicit `return()`. Personally I use `return()` only when I need to return early.

Reply:
@LittleBeannie Do you have a preference here? As I said, I'm totally fine with reverting to the original style `ans <- ...; return(ans)`, even though both the assignment and the explicit `return()` look unnecessary to me.

Reply:
Thank you for verifying, @yihui ! I personally prefer the explicit return, as it provides a clearer message for non-advanced R programmers.

Reply:
Got it. I've restored `return()` in 90352a04ad6d367b65777fd3915b5dc1ba5827fb.